get_notebook.py

```diff
@@ -0,0 +1,8 @@
+import os
+import galaxy
+import shutil
+
+hid = os.environ.get('DATASET_HID', None)
+if hid not in ('None', None):
+    galaxy.get(int(hid))
+    shutil.copy('/import/%s' % hid, '/import/ipython_galaxy_notebook.ipynb')
```

Comment:
Ok, sounds good. It just seemed strange to have so many checks when we could have had fewer since we have full control over the galaxy side and the container/notebook side.